lektor/imagetools.py

```diff
@@ -407,6 +406,12 @@ def read_exif(fp):
     return EXIFInfo(exif)
 
 
+def get_is_rotated_fast(fp):
```

Comment:
This is only nitpicking: I'd skip the "fast" in the name, since it's a specialised function and there's no "slow" version of it. And the docstring makes it clear enough why it's faster.

Reply:
Renamed as you suggested.